tests/lib/rules/no-classic-components.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+const rule = require('../../../lib/rules/no-classic-components');
+const RuleTester = require('eslint').RuleTester;
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+const { ERROR_MESSAGE } = rule;
+const ruleTester = new RuleTester();
+const parserOptions = {
```

Comment:
You can just pass `parserOptions` to `new RuleTester();` instead of including it with each test case.